src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -206,6 +267,37 @@ private static AssignmentExpr assignClientVariableWithCreateMethodExpr(TypeNode
         .build();
   }
 
+  // Create request variable by set attributes.
+  // e.g. EchoRequest request = EchoRequest.newBuilder().setParent(parent).build();
+  private static Expr createRequestBuilderExpr(
+      TypeNode requestType, List<MethodArgument> arguments) {
+    MethodInvocationExpr newBuilderExpr =
+        MethodInvocationExpr.builder()
+            .setStaticReferenceType(requestType)
+            .setMethodName("newBuilder")
+            .build();
+    for (MethodArgument arg : arguments) {
+      newBuilderExpr =
+          MethodInvocationExpr.builder()
+              .setExprReferenceExpr(newBuilderExpr)
+              .setMethodName(String.format("set%s", JavaStyle.toUpperCamelCase(arg.name())))
+              .setArguments(
+                  VariableExpr.withVariable(
+                      Variable.builder().setName(arg.name()).setType(arg.type()).build()))
+              .build();
+    }
+    MethodInvocationExpr requestBuildExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(newBuilderExpr)
+            .setMethodName("build")
+            .setReturnType(requestType)
+            .build();
+    return AssignmentExpr.builder()
+        .setVariableExpr(createVariableDeclExpr("request", requestType))
+        .setValueExpr(requestBuildExpr)
+        .build();
+  }
+
   private static Expr assignMethodArgumentWithDefaultValue(
```

Comment:
Can we refactor this so that we use the variable expr throughout (like one would do with Java programming) instead of recreating it several times?